src/user/serializers.py

```diff
@@ -1135,82 +1135,51 @@ def get_open_access_pct(self, author):
             )
 
     def get_reputation(self, author):
+        score = Score.objects.filter(author=author).order_by("-score").first()
+
+        if score is None:
+            return {}
```

Comment:
@yattias should I be returning an empty dict when score isn't calculated?

Reply:
This is the primary hub, right?  Better to return None